x-pack/legacy/plugins/uptime/public/state/actions/dynamic_settings.ts

```diff
@@ -6,7 +6,7 @@
 import { createAction } from 'redux-actions';
 import { DynamicSettings } from '../../../common/runtime_types';
 
-export const getDynamicSettings = createAction<{}>('GET_DYNAMIC_SETTINGS');
```

Comment:
Interesting, IIRC having no generic parameters caused me issues in the past, but it looks like it's resolved here!